src/app/donation-start/donation-start-form/donation-tipping-slider/donation-tipping-slider.component.ts

```diff
@@ -208,9 +222,19 @@ export class DonationTippingSliderComponent implements OnInit, AfterContentInit,
     this.handle.nativeElement.style.marginLeft = this.position + 'px';
   }
 
+  private detectWidth() {
+    this.width = this.bar.nativeElement.getBoundingClientRect().width - this.handle.nativeElement.getBoundingClientRect().width;
+  }
+
   updateHandlePositionFromClick(pageX: number) {
-    const mousePos = pageX - this.bar.nativeElement.offsetLeft - this.handle.nativeElement.offsetWidth / 2;
```

Comment:
The source of the bug was that offsetLeft is the offset from the containing positioned element, not the viewport edge, bug pageX was measured from the viewport width. Replacing it with `this.bar.nativeElement.getBoundingClientRect().width` fixed the bug.